src/inventory_ui.cpp

```diff
@@ -316,7 +316,8 @@ std::string inventory_selector_preset::get_cell_text( const inventory_entry &ent
     } else if( entry.is_item() ) {
         std::string text = cells[cell_index].get_text( entry );
         const item &actual_item = *entry.locations.front();
-        if( entry.get_category_ptr()->get_id() == item_category_id( "ITEMS_WORN" ) &&
+        if( cell_index == 0 && !text.empty() &&
+            entry.get_category_ptr()->get_id() == item_category_id( "ITEMS_WORN" ) &&
```

Comment:
Items are all displayed under the "ITEMS_WORN" category in the alternative inventory view (toggle with `;`). You might want to actually check that an item is worn by the player.

Reply:
I'll test that thanks.

Reply:
Thanks for the insight it would have took me a long time to figure this out on my own.

Reply:
Fixed with 0cc290a